roles/node/tasks/100-tests.yml

```diff
@@ -5,6 +5,12 @@
     msg: "The 'node_binary' variable can't be empty!"
   when: node_binary == ''
 
+# Enable to force GPG signature verification
+#- name: Test | Check node_binary_signature
+#  ansible.builtin.fail:
+#    msg: "The 'node_binary_signature' variable can't be empty and must be a URL!"
+#  when: node_binary_signature == '' or not node_binary_signature.startswith('http')
```

Comment:
Why do we need it in general? A user can use the role to deploy any own custom builds.